modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/PlaybackSettingsFragment.kt

```diff
@@ -677,19 +720,23 @@ class PlaybackSettingsFragment : BaseFragment() {
 }
 
 private enum class SettingsItems {
-    SETTINGS_GENERAL_DEFAULTS,
-    SETTINGS_GENERAL_UP_NEXT,
-    SETTINGS_GENERAL_EPISODE,
-    SETTINGS_GENERAL_ARCHIVED_EPISODES,
+    SETTINGS_HEADER_DEFAULTS,
+    SETTINGS_ROW_ACTION,
+    SETTINGS_UP_NEXT_SWIPE,
+    SETTINGS_EPISODE_GROUPING,
+    SETTINGS_ARCHIVED_EPISODES,
     SETTINGS_MEDIA_NOTIFICATION_CONTROLS,
-    SETTINGS_GENERAL_PLAYER,
+    SETTINGS_HEADER_PLAYER,
+    SETTINGS_SKIP_FORWARD_TIME,
     SETTINGS_SKIP_BACK_TIME,
-    SETTINGS_GENERAL_KEEP_SCREEN_AWAKE,
-    SETTINGS_GENERAL_OPEN_PLAYER_AUTOMATICALLY,
-    SETTINGS_GENERAL_INTELLIGENT_PLAYBACK,
-    SETTINGS_GENERAL_PLAY_UP_NEXT,
-    SETTINGS_USE_REAL_TIME_FOR_PLAYBACK_REMAINING_TIME,
-    SETTINGS_GENERAL_SLEEP_TIMER,
+    SETTINGS_KEEP_SCREEN_AWAKE,
+    SETTINGS_OPEN_PLAYER_AUTOMATICALLY,
+    SETTINGS_INTELLIGENT_PLAYBACK,
+    SETTINGS_PLAY_UP_NEXT_EPISODE,
+    SETTINGS_ADJUST_REMAINING_TIME,
+    SETTINGS_HEADER_SLEEP_TIMER,
+    SETTINGS_SLEEP_TIMER_RESTART,
+    SETTINGS_SLEEP_TIMER_SHAKE,
```

Comment:
I renamed some of these to make the names shorter and also split some that had grouped settings so they all matched.